### PR TITLE
[DOC-474] Document upgrading via Homebrew, along with caveats

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-474] Document upgrading via Homebrew, along with caveats

### CODE REVIEW

**Review by dbist — APPROVED** [review]
LGTM

File: src/current/v22.2/install-cockroachdb-mac.md [hunk 1]
```diff
@@ -52,9 +52,22 @@ Use one of the options below to install CockroachDB.
         {% include marketo-install.html uid="1" %}
     </li>
   </ol>
-{{site.data.alerts.callout_info}}
-If you previously installed CockroachDB via Homebrew, run <code>brew uninstall cockroach</code> before installing the new version. If you installed using a different method, you may need to remove the binary before installing via Homebrew.
-{{site.data.alerts.end}}
+{{site.data.alerts.callout_success}}
+If you previously installed CockroachDB via Homebrew, you can [upgrade]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}) the CockroachDB binary to the next major version or to a patch version using HomeBrew. After updating the binary on each node, restart the `cockroach` process on the node. When upgrading to a new major version, you must complete additional steps to [finalize]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#step-6-finish-the-upgrade) the upgrade. If you need to upgrade through multiple major versions, you must complete each major-version upgrade separately, including finalizing the upgrade, before beginning the next one.
```
Comment (kathancox) [inline src/current/v22.2/install-cockroachdb-mac.md:56]: The links are not rendering in each of the versions. I'm not sure why? In v22.2, I think we want to use the old link format... however the rendering issue is also present in v23.1 & v23.2

Comment (kathancox) [inline src/current/v22.2/install-cockroachdb-mac.md:56]: The inline code here "`cockroach`" is also not rendering properly.

File: src/current/v22.2/install-cockroachdb-mac.md [hunk 2]
```diff
@@ -52,9 +52,22 @@ Use one of the options below to install CockroachDB.
...
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+brew update
+~~~
+~~~ shell
+brew update
+brew upgrade cockroach
+~~~
+{site.data.alerts.end}}
```
Comment (kathancox) [inline src/current/v22.2/install-cockroachdb-mac.md:70]: Ah the "{site.data.alerts.end}}" is missing a curly brace and is visible in the preview.

**Review by kathancox — COMMENTED** [review]
There's a rendering issue in each of the versions here for the link, inline code, and the tip box liquid at the end.

**Comment by mdlinville** [comment]
> There's a rendering issue in each of the versions here for the link, inline code, and the tip box liquid at the end.

All of these issues were caused by omitting `markdown="1"` from the filter-tab divs. Confirmed that all three versions are now rendering correctly.

**Pull request merged**